### DISCUSSION TITLE
Belt Clip mounts (Tape Measures, Knives, etc)

### DISCUSSION THREAD

**Discussion by razorfrog** [aderusha/DDD-Printable-Wall-Control-System#61] [Q&A] [answered]
Hi! I'm just getting started with this system and it's very exciting stuff. I've figured out a few recipes for tools so far, but I'm not sure about the tape measures and razor knives in your photos. How are they attached? Is that just a blank spacer? I wouldn't think there's enough room behind it. Thanks!

<img width="1306" alt="Screenshot 2023-01-24 at 9 22 56 PM" src="https://user-images.githubusercontent.com/1851688/214486462-de1840cd-0c6a-4501-a074-80404030ea0f.png">

<img width="892" alt="Screenshot 2023-01-24 at 9 24 19 PM" src="https://user-images.githubusercontent.com/1851688/214486635-376c2bec-2d57-4b85-8ee7-2a7145cf1925.png">

**Comment by razorfrog** [comment]
Aw jeeze, it's the **Spacer_clip-on** folder isn't it. I'll print some of those tonight.

**Reply by root-hal9000** [reply]
I came here to get some of those to print out. Some of my clip on stuff seems to tight to work, but I am about to find out too!

**Comment by razorfrog** [comment, marked as answer]
Works well! I did this: 
- 1× 2x3 spacer clip-on
- 1× 1x3 locking spacer
- 1x 3x0 flat right
- 1× 3x0 flat left
- 4× 4x10x8mm pin
- 1× 8mm lock pin
